kano_video/ui/video.py

```diff
@@ -43,6 +43,7 @@ class VideoEntry(Gtk.Button):
 
     def __init__(self, e, playlist_name=None, permanent=False):
         super(VideoEntry, self).__init__(hexpand=True)
+        stats_str = ""
```

Comment:
It looks like the only times that this [is referenced below](https://github.com/cwilson-github/kano-video/blob/patch-3/kano_video/ui/video.py#L88-L91) is when it is defined immediately beforehand. As such, would I be correct in saying that this line is unnecessary?